section4/exercises/classMethods.js

```diff
@@ -28,13 +28,29 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
-
-  // ADD CODE
+changeProtein(newProtein) {
+  this.protein = newProtein;
+}
+addTopping(newTopping) {
+  this.toppings.push(newTopping);
```

Comment:
Great work finding the method to get the job done. When researching methods, remember experimentation is very helpful.